chapters/01/fundamentals.md

```diff
@@ -27,18 +27,6 @@
 jinja is web template engine for Python that uses curly brackets to define it's variables.
 ```
 
-```python
-{%- for item in item_list %}
-    <li>{{ item }}</li>
-{%- endfor %}
-```
-
-```html
-<li>1</li>
-<li>2</li>
-<li>3</li>
-```
-
```

Comment:
Btw, why was this left out? This was added based on Kada's comment that he doesn't know jinja.

Reply:
Oh, okay 😅  I thought it was added by an accident. I will leave it there